src/components/Kv/KvLoadingParagraph.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="loading-paragraph">
+		<kv-loading-placeholder v-for="n in numLines" :key="n" />
+	</div>
+</template>
+
+<script>
+import KvLoadingPlaceholder from '@/components/Kv/KvLoadingPlaceholder';
+import getCacheKey from '@/util/getCacheKey';
+
+export default {
+	name: 'KvLoadingParagraph',
+	props: {
+		numLines: {
+			type: Number,
+			default: 4,
+		},
+	},
+	serverCacheKey: ({ numLines }) => getCacheKey(`KvLoadingParagraph-${numLines}`),
+	components: {
+		KvLoadingPlaceholder,
+	},
+};
+</script>
+
+<style lang="scss">
```

Comment:
Can we keep this scoped as well as the styles in KvLoadingPlaceholder? In this template you should be able to add a class like `<kv-loading-placeholder class="placeholder"> `and set the same styles on `.placeholder` directly.

Reply:
👍

Reply:
I think we should figure out how we'd like to handle managing style scope as a team at some point soon.